charts/redpanda/values.yaml

```diff
@@ -87,6 +87,26 @@ license_secret_ref: {}
   # secret_name: my-secret
   # secret_key: key-where-license-is-stored
 
+# -- Audit logging for a redpanda cluster, must have enabled sasl and have one kafka listener supporting sasl authentication
+#    for audit logging to work. Note this feature is only available for redpanda versions >= v23.3.0
+auditLogging:
+  # -- Enable or disable audit logging, for production clusters we suggest you enable.
+  #    however, this will only work if you also enable sasl and a listener with sasl enabled.
+  enabled: true
+  # -- Kafka external listener name, note that it must have `authenticationMethod` set to sasl
+  listener: default
+  # -- Integer value defining the number of partitions used by a newly created audit topic
+  partitions: 12
+  # -- Integer valued specifies the replication factor for a newly created audit log topic
+  logReplicationFactor: 3
```

Comment:
Not sure we want to add this.  By default, if this config is not set, the audit log system will use `internal_topic_replication_factor` to determine it's RF.  By default that is 3.  What's helpful about using `internal_topic_replication_factor` is that if the cluster is only one node, the value auto-updates to 1.  By hard setting this to 3, the topic will attempt to be created with an RF of 3 which will fail on a single node cluster.

My suggestion is to remove this.